apollo-backend/src/test/java/io/logz/apollo/AuthTest.java

```diff
@@ -121,7 +121,7 @@ public void testDeploymentWithNoPermissions() throws ApolloClientException {
         testDeployableVersion.setId(apolloTestClient.addDeployableVersion(testDeployableVersion).getId());
 
         Deployment testDeployment = ModelsGenerator.createDeployment(testService, testEnvironment, testDeployableVersion);
-        assertThatThrownBy(() -> apolloTestClient.addDeployment(testDeployment)).isInstanceOf(ApolloNotAuthorizedException.class);
+        assertThat(apolloTestClient.addDeployment(testDeployment).getUnsuccessful().get(0).get("reason")).isEqualTo("Not Authorized!");
```

Comment:
Im not sure i like this change.
Now if im using the Java client, i need to be aware of the wording of the exceptions instead of the actual exceptions.
Maybe add the actual exception to the response object?
then we can have .getUnsuccessful().get(0).getException()